frontend/ui/widget/touchmenu.lua

```diff
@@ -95,6 +95,8 @@ function TouchMenuItem:init()
         checked = true,
     }
 
+    self.checkmark_tap_width = checked_widget:getSize().w + 2*Size.padding.default
```

Comment:
Well, it's a new UX feature. I think I rememebr a couple of places where it can be used.